src/styles.scss

```diff
@@ -50,6 +50,12 @@
 @import "./components/TextInput/_TextInput.scss";
 @import "./components/Autosuggest/_Autosuggest.scss";
 
+fieldset {
```

Comment:
This is temporary but we need better global styles for these rules.

Reply:
Hmm. Let's talk about this.